pkgs/tools/networking/acme-client/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, apple_sdk
+, cacert
+, defaultCaFile ? "${cacert}/etc/ssl/certs/ca-bundle.crt"
+, fetchurl
+, libbsd
+, libressl
+, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  name = "acme-client-${version}";
+  version = "0.1.16";
+
+  src = fetchurl {
+    url = "https://kristaps.bsd.lv/acme-client/snapshots/acme-client-portable-${version}.tgz";
+    sha512 = "730c20bdf9d72b24e66c54b009a282e04da3ea8ce3b9eb053750672c53c9586b2879d87a565ddbab033d7ba6a577dd6399313b20cf654b185905db4de988b6b7";
+  };
+
+  buildInputs = [ libbsd libressl pkgconfig ]
+    ++ stdenv.lib.optional stdenv.isDarwin apple_sdk.sdk;
+
+  CFLAGS = "-DDEFAULT_CA_FILE='\"${defaultCaFile}\"'";
+
+  preConfigure = ''
+    export PREFIX="$out"
+  '';
+
+  meta = {
+    homepage = https://kristaps.bsd.lv/acme-client/;
+    description = "Secure acme/Let's Encrypt client";
+    platforms = stdenv.lib.platforms.all;
```

Comment:
If `meta` had `with stdenv.lib`, it could have been omitted on individual attributes:

```
meta = with stdenv.lib; {
  platforms = platforms.all;
  license = licenses.isc;
}
```

Also, it seems unlikely to me that it would run on Windows: probably you've meant `platforms.unix`.

Reply:
Great review, appreciate the careful look! Will make all those changes.